src/src/service/user_service.js

```diff
@@ -1,3 +1,7 @@
 import { createContext } from 'react';
 
 export const UserContext = createContext('Default Value');
+
+
+export const DATA_ADMIN = "89a69625-99d7-11ea-9366-0e98982705c1";
+export const DATA_CURATOR = "75804b96-d4a8-11e9-9da9-0ad4acb67ed4";
```

Comment:
@BirdMachine I realized that we have been hardcoding these config values in the js modules. Can we move them to the the config file instead? Especially when we have HuBMAP Read group uuid and the Data-Admin group name already specified there: https://github.com/hubmapconsortium/ingest-ui/blob/main/src/example.env#L20-L24